src/mantarray_desktop_app/log_formatter.py

```diff
@@ -13,13 +14,14 @@ class SensitiveFormatter(logging.Formatter):
     """
 
     @staticmethod
-    def _filter(log_msg: str) -> str:
-        return re.sub(
-            r"(.*set_mantarray_nickname\?nickname=)(.*)( HTTP.*)",
-            lambda match_obj: match_obj[1] + get_redacted_string(len(match_obj[2])) + match_obj[3],
-            log_msg,
-        )
+    def _filter(log_msg: str) -> Any:
+        if "/update_settings" not in log_msg:
```

Comment:
Why did this need to be added? I wouldn't think nicknames would ever show up in /update_settings

Reply:
@tannermpeterson I added that line so that the request with customer creds/password doesn't get logged. Doesn't have to do with the nickname.